docs/sources/dashboards/manage-dashboards/index.md

```diff
@@ -32,59 +32,61 @@ weight: 8
 
 # Manage dashboards
 
-A dashboard is a set of one or more [panels][] that visually presents your data in one or more rows.
-
-For more information about creating dashboards, refer to [Add and organize panels](../add-organize-panels).
-
 This topic includes techniques you can use to manage your Grafana dashboards, including:
 
 - [Creating and managing dashboard folders](#create-and-manage-dashboard-folders)
 - [Exporting and importing dashboards](#export-and-import-dashboards)
 - [Organizing dashboards](#organize-a-dashboard)
 - [Troubleshooting dashboards](#troubleshoot-dashboards)
 
+For more information about creating dashboards, refer to [Add and organize panels](../add-organize-panels).
+
+## Browse dashboards
+
+On the Dashboards page, you can browse and manage folders and dashboards, including:
+
+- Creating folders and dashboards
+- Move dashboards between folders
+- Delete multiple dashboards and folders
+- Navigate to a folder
+- Manage folder permissions. For more information, see [Dashboard permissions](https://grafana.com/docs/grafana/latest/administration/roles-and-permissions/#dashboard-permissions)
+
+{{% admonition type="note" %}}
+As of Grafana 10.2 there is no longer the special 'General' folder. Dashboards without a folder are now shown at the top level along side folders.
+{{% /admonition %}}
+
 ## Create a dashboard folder
 
 Folders help you organize and group dashboards, which is useful when you have many dashboards or multiple teams using the same Grafana instance.
 
-**Before you begin:**
-
-- Ensure that you have Grafana Admin or Super Admin permissions. For more information about dashboard permissions, refer to [Dashboard permissions][].
+> **Before you begin:** Ensure you have Editor or greater permission to create folders. For more information about dashboard permissions, refer to [Dashboard permissions][].
 
 **To create a dashboard folder:**
 
-1. Sign in to Grafana.
 1. Click **Dashboards** in the left-side menu.
 1. On the Dashboards page, click **New** and select **New folder** in the dropdown.
 1. Enter a unique name and click **Create**.
 
-   Do not use the name of a folder that has already been provisioned (such as "General") and avoid special characters (except underscores and hyphens).
```

Comment:
I know we don't need the warning about the General folder, but are special characters okay now?

Reply:
Maybe my understanding of what 'special characters' are is limited, but I'm not aware of such a restriction. To be honest, I'm not sure if there ever was a restriction on this...

<img width="763" alt="Screenshot 2023-10-09 at 10 56 38 am" src="https://github.com/grafana/grafana/assets/46142/8e20c7ce-9a1d-42f6-80a7-85f2b797fd18">

Reply:
This had come up on a [support ticket](https://github.com/grafana/support-escalations/issues/5706). There was specifically an issue with folks putting slashes in folder names. So maybe that's the only limitation?

Reply:
Ahhh - good catch. Yes, I believe there is a limitation with Alerting is that _Alerts_ cannot be placed in folders with slashes in them. It would be good to mention that here.

<img width="451" alt="Screenshot 2023-10-10 at 3 18 59 pm" src="https://github.com/grafana/grafana/assets/46142/3cacb432-acb5-4345-8551-d99293d570b3">